### PR TITLE
fix: Add VCS to check title

### PR DESCRIPTION
**Pull request by tsmithv11**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

[//]: # "
    # PR Title
    Be aware that we use the title to create changelog automatically and therefore only allow specific prefixes
    - break:    to indicate a breaking change, this supersedes any of the types
    - feat:     to indicate new features or checks
    - fix:      to indicate a bugfix or handling of edge cases of existing checks
    - docs:     to indicate an update to our documentation
    - chore:    to indicate adjustments to workflow files or dependency updates
    - platform: to indicate a change needed for the platform
    Additionally we encourage adding a scope to the prefix, which indicates the targeted framework, otherwise 'general' will be assumed.
    ex.
    feat(terraform): add CKV_AWS_123 to ensure that VPC Endpoint Service is configured for Manual Acceptance
"

## Description

Updating Terraform VCS provider checks to be more descriptive

### CODE REVIEW

**Review by gruebel — APPROVED** [review]
🍰

File: checkov/terraform/checks/resource/gitlab/ForcePushDisabled.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 class ForcePushDisabled(BaseResourceValueCheck):
     def __init__(self) -> None:
-        name = "Ensure force push is disabled"
+        name = "Ensure GitLab branch protection rules does not allow force pushes"
```
Comment (mikeurbanski1) [inline checkov/terraform/checks/resource/gitlab/ForcePushDisabled.py:9]:
```suggestion
        name = "Ensure Gitlab branch protection rules does not allow force pushes"
```

File: checkov/terraform/checks/resource/gitlab/PreventSecretsEnabled.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 class PreventSecretsEnabled(BaseResourceValueCheck):
     def __init__(self) -> None:
-        name = "Ensure prevent secrets is enabled"
+        name = "Ensure GitLab prevent secrets is enabled"
```
Comment (mikeurbanski1) [inline checkov/terraform/checks/resource/gitlab/PreventSecretsEnabled.py:9]:
```suggestion
        name = "Ensure Gitlab prevent secrets is enabled"
```

File: checkov/terraform/checks/resource/gitlab/RejectUnsignedCommits.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 class RejectUnsignedCommits(BaseResourceValueCheck):
     def __init__(self) -> None:
-        name = "Ensure commits are signed"
+        name = "Ensure GitLab commits are signed"
```
Comment (mikeurbanski1) [inline checkov/terraform/checks/resource/gitlab/RejectUnsignedCommits.py:9]:
```suggestion
        name = "Ensure Gitlab commits are signed"
```

File: checkov/terraform/checks/resource/gitlab/RequireTwoApprovalsToMerge.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
 class RequireTwoApprovalsToMerge(BaseResourceCheck):
     def __init__(self) -> None:
-        name = "Ensure at least two approving reviews to merge"
+        name = "Ensure at least two approving reviews are required to merge a GitLab MR"
```
Comment (mikeurbanski1) [inline checkov/terraform/checks/resource/gitlab/RequireTwoApprovalsToMerge.py:12]:
```suggestion
        name = "Ensure at least two approving reviews are required to merge a Gitlab MR"
```

**Review by mikeurbanski1 — APPROVED** [review]

**Pull request merged**